frontends/web/src/components/devices/bitbox02/backups.tsx

```diff
@@ -101,67 +102,70 @@ class Backups extends Component<Props, State> {
-                                            onFocus={() => undefined}
-                                            radio={showRadio} />
-                                    </table>
-                                )) }
+                        errorText && (
+                            <div className={style.standOut}>
+                                <img src={alertOctagon} />
+                                <span className={style.error}>{errorText}</span>
                             </div>
-                        ) :
-                        (
-                            <p>
-                                {t('backup.noBackups')}
-                            </p>
-                        )
-                    }
-                </div>
-                <div class="buttons bottom flex flex-row flex-between">
-                    {children}
-                    {
-                        showRestore && (
-                            <Button
-                                primary={true}
-                                disabled={!selectedBackup || restoring}
-                                onClick={this.restore}>
-                                {t('button.restore')}
-                            </Button>
-                        )
-                    }
-                    {
-                        showCreate && (
-                            <Check
-                                deviceID={deviceID}
-                                backups={backups.backups ? backups.backups : []}
-                                disabled={backups.backups!.length === 0}
-                            />
-                        )
-                    }
-                    {
-                        showCreate && (
-                            <Create deviceID={deviceID} />
                         )
                     }
+                    <div class={backupStyle.backupsList}>
+                        {
+                            backups.backups!.length ? (
+                                <div>
+                                    {/* <SimpleMarkup tagName="p" markup={t('backup.list')} /> */}
+                                    <p>The following backups have been found. Please select which backup to restore.</p>
+                                    <div className={backupStyle.listContainer}>
+                                        {
+                                            backups.backups!.map(backup => (
+                                                <table className={style.table}>
+                                                    <BackupsListItem
+                                                        key={backup.id}
+                                                        disabled={restoring}
+                                                        backup={backup}
+                                                        selectedBackup={selectedBackup}
```

Comment:
Do we still need to select backups? Check Backup go through the entire list of backups and checks if at least one of them matches the current wallet. Create Backup just creates a new one from the current wallet, so you do not need to select either.

Reply:
i think this might be reused with when you restore from backup, where you need to select one.